app/lib/ckan/v26/depaginator.rb

```diff
@@ -18,60 +17,63 @@ def initialize(base_url, existing_total:)
       def depaginate
```

Comment:
General comment: there's a bit much nesting in this function; I found it quite hard to read (especially the diff). It was already like that though so I get it if you don't fancy refactoring it right now.